src/browser/api/window.js

```diff
@@ -1708,22 +1708,26 @@ function getBoundsCacheSafeFileName(identity) {
 
 function applyAdditionalOptionsToWindowOnVisible(browserWindow, callback) {
     if (browserWindow.isVisible()) {
-        callback();
+        readySetGo();
```

Comment:
Actually, you can ignore this change. This condition, `if (browserWindow.isVisible())`, was only added because we reuse this function to reapply visibility-sensitive features, like rounded corners and resize regions, for `updateOptions()`. On window creation, which is the else statement, is the only point we care about `ready-to-show`.

Reply:
So I think you're saying leave the conditional only call `callback` instead of `readSetGo`, right?

Reply:
right